defaults/main.yml

```diff
@@ -18,6 +18,10 @@ wimpy_application_pre_commands: []
 wimpy_application_post_commands: []
 wimpy_application_additional_units: []
 
+# CloudWatch logs
+wimpy_aws_cloudwatch_enabled: True
```

Comment:
Ok for me but I'd put in README an example of "lightweight-savemoney" configuration :P